### ISSUE TITLE
Fix T006 FN: Raise on namespaces

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#9668]
The rule should also raise on namespace declarations
```
namespace SonarAnalyzer.Helpers;
```
and
```
namespace SonarAnalyzer.Helpers
{
}
```

**Comment by pavel-mikula-sonarsource** [comment]
https://sonarsource.atlassian.net/browse/NET-841
